Fabric.Terminology.API/Scripts/Terminology-Intall-Utilities.psm1

```diff
@@ -0,0 +1,135 @@
+Class TerminologyConfig
+{
+    [String]$siteName
+    [String]$appName
+    [String]$sqlServerAddress
+    [String]$appPool
+    [String]$applicationEndpoint
+    [String]$discoveryServiceUrl
+    [PSCredential]$iisUserCredentials
+}
+function Import-Modules() {
+    # Dos Install Utilities
+    $dosInstallUtilities = Get-Childitem -Path ./**/DosInstallUtilities.psm1 -Recurse
+    if($dosInstallUtilities.length -eq 0) {
+        Write-Warning "could not find dos install utilities. Manually installing "
+        Install-Module DosInstallUtilities -Scope CurrentUser
+        Import-Module DosInstallUtilities -Force
+    } else {
+        Import-Module -Name $dosInstallUtilities.FullName -Force
+    }
+	
+	# Fabric Install Utilities
+	$fabricInstallUtilities = ".\Fabric-Install-Utilities.psm1"
+    if(!(Test-Path $fabricInstallUtilities -PathType Leaf)){
+        Invoke-WebRequest -Uri https://raw.githubusercontent.com/HealthCatalyst/InstallScripts/master/common/Fabric-Install-Utilities.psm1 -Headers @{"Cache-Control"="no-cache"} -OutFile $dosInstallUtilities
```

Comment:
Think the outfile name is wrong here - it looks like you might overwrite the dos installutilities module?

Reply:
good catch!